frontend/scss/homepage/_main.scss

```diff
@@ -10,26 +10,30 @@
 
   .hero-banner-content {
     max-width: $context-width;
-    margin: 0 auto;
-    padding: 70px 15px;
     color: white;
+    position: relative;
 
-    h1, p {
-      max-width: 700px;
-    }
     h1 {
-      font-size: 3rem;
+      font-size: 2rem;
+      font-family: Trebuchet MS;
       color: white;
+      background: rgba(0, 0, 0, 0.5);
+      padding: 80px;
+      max-width: 500px;
```

Comment:
@Sharonsyra not looking good at on smaller screens i.e @media screen and (max-width: 992px)
.hero-banner .hero-banner-content h1